benchmarks/run_all.py

```diff
@@ -48,7 +48,7 @@ def main(args, rank):
         ops_to_run.append('all_to_all')
 
     if len(ops_to_run) == 0:
-        ops_to_run = ['all_reduce', 'all_gather', 'all_to_all', 'broadcast', 'pt2pt']
+        ops_to_run = ['all_reduce', 'all_to_all', 'broadcast', 'pt2pt', 'all_gather']
```

Comment:
why reorder these?

Reply:
I will revert this change. For testing purpose I did this change as all_gather was failing in case of mvapich-plus so I wanted to test all other collectives before all_gather.